e2e/bitrise.yml

```diff
@@ -181,37 +219,28 @@ workflows:
         is_always_run: true
         inputs:
         - envs:
-        - files:
+        - files: |-
+            BITRISE_XCTESTRUN_FILE_PATH
```

Comment:
the xctestrun file is not moved to the deploy dir anymore, as it needs the whole bundle to be in the same directory (xctestrun file + built test dir)